### ISSUE TITLE
Dependency Issues

### ISSUE THREAD

**Issue by bpappin** [davidmoten/geo#32]
Looks like there is a dependency reference on java.awt in the library.
Although this isn't an Android library, it might be handy to clean up this kind of reference.

`Package not included in Android
../../../../../.gradle/caches/modules-2/files-2.1/org.apache.commons/commons-math3/3.2/ec2544ab27e110d2d431bdad7d538ed509b21e62/commons-math3-3.2.jar: Invalid package reference in library; not included in Android: java.awt.geom. Referenced from org.apache.commons.math3.geometry.euclidean.threed.PolyhedronsSet.RotationTransform.
../../../../../.gradle/caches/modules-2/files-2.1/com.github.davidmoten/grumpy-core/0.2.2/48b4cff02a62a3a9ea876dd2486225aefd225706/grumpy-core-0.2.2.jar: Invalid package reference in library; not included in Android: java.awt. Referenced from com.github.davidmoten.grumpy.core.Position.`

**Comment by bpappin** [comment]
It might be enough to ignore the awt package in the ProGuard config.

**Comment by davidmoten** [comment]
I agree that it would be nice to remove the reference to awt. That reference is in the *grumpy* dependency. I'll have a look at removing it in a week or two (once back from holidays).

**Comment by bhandfast** [comment]
Had any chance on looking at that dependency? Seems geo only uses a Position class from grumpy. Maybe it's possible to copy that implementation instead?

Thanks for a nice library 👍

**Comment by SebastianOltmanns** [comment]
Experiencing the same problem. Please reconsider fixing this.

**Comment by davidmoten** [comment]
Having a look now

**Comment by davidmoten** [comment]
I've removed dependency on grumpy-core. Release with fix #37 is on Maven Central now, version 0.7.7.
